src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -843,12 +845,14 @@ def _validate_cmd_level(self, ns, cmd_validator):  # pylint: disable=no-self-use
             pass
 
     def _validate_arg_level(self, ns, **_):  # pylint: disable=no-self-use
-        from msrest.exceptions import ValidationError
         for validator in getattr(ns, '_argument_validators', []):
             try:
                 validator(**self._build_kwargs(validator, ns))
-            except ValidationError:
-                logger.debug('Validation error in %s.', str(validator))
+            except Exception as ex:
+                # Delay the import and mimic an exception handler
+                from msrest.exceptions import ValidationError
+                if isinstance(ex, ValidationError):
+                    logger.debug('Validation error in %s.', str(validator))
```

Comment:
Mimic the behavior of https://github.com/Azure/azure-cli/blob/e6e437a80639616c9871d1d9248bbb1a65be268e/src/azure-cli-core/azure/cli/core/util.py#L58-L63